R/addChart.R

```diff
@@ -10,7 +10,7 @@
 #' @param repo_url Homepage for chart's code repository (if any)
 #' @param settings_url Homepage for chart's settings documentation
 #' @param main Name of the main function used to initialize the app. If the type is htmlwidgets, the js function must accept "location" and "settings" parameters (in that order) and have an .init() method, expecting a json data array. Otherwise, the r function should accept named data and settings parameters, and should be loaded in the user's namespace.
-#' @param type type of chart. Should be 'htmlwidget', 'static', 'plotly' or 'module'
```

Comment:
Should we support in v1.1? Or hold off for 1.2?

Reply:
Based on our conversations, I think this is complex enough and we should wait. Ideally, we'd move the safety renderers to a separate repo (and make that a dependency) as part of that update.